keps/sig-node/693-topology-manager/README.md

```diff
@@ -594,49 +593,307 @@ feature gate and set the CPU Manager policy to 'static'.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+"topology_manager_admission_duration_seconds" (which will be added as this release) can be used to determine
+if the resource alignment logic performed at pod admission time is taking longer than expected.
+
+Measurements haven't been performed to determine the latency as this metric will be introduced in 1.27
+development cycle but the duration is expected to be very short most likely in the ballpark of 50-100 ms.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [X] Metrics
+  - Metric name:
+    - topology_manager_admission_requests_total
+    - topology_manager_admission_errors_total
+    - topology_manager_admission_duration_seconds
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No.
+
+### Dependencies
+
+No.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No.
+
+### Scalability
+
+Topology Manager is a component within kubelet for alignment of resources
+based on resource distribution across NUMA nodes and configured policy.
+Since this is a node-local feature, there are no calls to the API
+server or to the cloud provider and hence does not impact scalability.
+
+###### Will enabling / using this feature result in any new API calls?
+
+No, this is a node-local feature.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No, this is a node-local feature.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No, this is a node-local feature.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No.
+
+This feature is not impacted by the scale of the cluster (number of nodes in the cluster) as that is not relevant and is not factored into the alignment algorithm. It is the scheduler that has to deal with the scalability aspect and determine nodes that can fulfill the resources requested by the pod. If this feature is turned off, the scheduler would still have to perform the same computation as it would if this feature was enabled. Hence, this feature is not impacted by scale or impacts the scalability of a cluster.
+
+Also, the resource alignment logic is executed at pod admission time which is prior to pod startup so there shouldn't be any impact on pod startup latency. The check at admission is to determine if the pod is suitable to be admitted on the node based on the configured policy. If considered suitable, the pod proceeds to startup (following which the pod startup latency is measured).
```

Comment:
nit: Shouldn't it be included since pod startup latency is measured from pod object creation time? 
That said the answer here could cover the numa limit of 8 to bound the time it takes in admission.

Reply:
I see, if the pod startup latency is measured from the time the pod objected is created then definitely this feature impacts the pod startup latency. I thought  that the pod startup latency is measured from the time time pod is started on the node to the point in time when it is up and running. I will update the KEP.

Reply:
Updated the KEP to addressed this comment. Thanks!